src/components/navigation/navbar/ModuleNavBar.tsx

```diff
@@ -17,67 +14,41 @@ export interface NavigationItem {
   const location = useLocation();
-  const [isOpen, setIsOpen] = useState(true);
 
   return (
-    <div
-      className={classNames(
-        'relative flex grow-0 flex-col gap-y-5 overflow-y-auto',
-        isOpen ? 'bg-white min-w-40' : 'bg-white min-w-14'
-      )}
-    >
-      {isOpen ? (
-        <>
-          <button
-            type='button'
-            onClick={() => setIsOpen((p) => !p)}
-            className='absolute right-0 top-0 m-4 rounded-md text-gray-400 hover:text-gray-500 focus:outline-none focus:ring-2 focus:ring-blue-500/50 focus:ring-offset-2'
-          >
-            <span className='absolute -inset-2.5' />
-            <span className='sr-only'>Close panel</span>
-            <XMarkIcon aria-hidden='true' className='h-6 w-6' />
-          </button>
-          <nav className='flex flex-1 flex-col'>
-            <ul role='list' className='flex flex-1 flex-col space-y-1 px-1'>
-              {navigation.map((item, index) => (
-                <li key={index}>
-                  {item.title ? (
-                    <div className='pt-3 pb-3 pl-5 pr-6 text-xl font-medium'>{item.title}</div>
-                  ) : (
-                    <div className='pb-3'></div>
+    <div className='flex grow-0 flex-col gap-y-5 overflow-y-auto bg-magpie-light-25 min-w-40 border-r-2 border-magpie-light-50 px-1'>
+      <nav className='flex flex-1 flex-col'>
+        <ul role='list' className='flex flex-1 flex-col gap-y-7 space-y-1 px-1'>
+          {navigation.map((item, index) => (
+            <li key={index}>
+              {item.title ? (
+                <div className='py-5 pl-5 pr-6 text-xl font-medium'>{item.title}</div>
+              ) : (
+                <div className='pb-3'></div>
+              )}
+
+              {item.children.map((item) => (
+                <Link
+                  key={item.name}
+                  to={item.href}
+                  className={classNames(
+                    'group flex gap-x-2 py-2 px-8 text-sm rounded-md leading-6 font-semibold text-magpie-dark-75',
+                    location.pathname.includes(item.href)
+                      ? 'bg-magpie-light-50'
+                      : 'hover:bg-magpie-light-50'
                   )}
-
-                  {item.children.map((item) => (
-                    <Link
-                      key={item.name}
-                      to={item.href}
-                      className={classNames(
-                        'group flex gap-x-3 py-2 px-10 text-sm rounded-md leading-6 font-normal text-magpie-dark-75 hover:bg-magpie-light-25',
-                        item.isCurrent || item.href == location.pathname ? 'bg-magpie-light-25' : ''
-                      )}
-                    >
-                      {item.name}
-                    </Link>
-                  ))}
-                </li>
+                >
+                  {item.name}
+                </Link>
               ))}
-            </ul>
-          </nav>
-
-          {footer && footer}
```

Comment:
I guess this file does not survive the merge conflict, but I used the footer for the library nav bar. I think the X-mark button my also be useful for smaller screen.

Reply:
sure, I will have a check